shared/teams/team/container.js

```diff
@@ -197,9 +197,13 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
     customComponent,
     headerStyle: {borderBottomWidth: 0},
     invites: stateProps._invites.toJS(),
-    members: stateProps._memberInfo
-      .toArray()
-      .sort((a: Constants.MemberInfo, b: Constants.MemberInfo) => a.username.localeCompare(b.username)),
+    members: stateProps._memberInfo.toArray().sort((a: Constants.MemberInfo, b: Constants.MemberInfo) => {
+      if (you && !youImplicitAdmin) {
```

Comment:
hmm, not sure if the `!youImplicitAdmin` is needed. If you're an implicit admin, you won't show up in the member list at all, right?